### PR TITLE
feat: Add support for additional_instructions

### PR DESCRIPTION
**Pull request by geoah**

### CODE REVIEW

File: ez_openai/__init__.py [hunk 1]
```diff
@@ -127,13 +130,16 @@ def ask(
         message: str | None,
         image_url: str | None = None,
         image_file: bytes | None = None,
+        additional_instructions: Optional[str] | NotGiven = NOT_GIVEN,
```
Comment (skorokithakis) [inline ez_openai/__init__.py]: How about this?:

```suggestion
        additional_instructions: str | None = None,
```

Comment (geoah) [reply ez_openai/__init__.py]: The OpenAI lib distinguishes between None and NotGiven and wasn't sure why.
```
additional_instructions: Optional[str] | NotGiven = NOT_GIVEN,
```

Comment (geoah) [reply ez_openai/__init__.py]: https://github.com/openai/openai-python/blob/6316b7c0f182877e578a3dbed2fd129b63538af5/src/openai/resources/beta/threads/runs/runs.py#L1653

File: ez_openai/__init__.py [hunk 2]
```diff
@@ -127,13 +129,16 @@ def ask(
         message: str | None,
         image_url: str | None = None,
         image_file: bytes | None = None,
+        additional_instructions: Optional[str] = None,
```
Comment (skorokithakis) [inline ez_openai/__init__.py]:
```suggestion
        additional_instructions: str | None = None,
```

to match the above.

File: ez_openai/__init__.py [hunk 3]
```diff
@@ -167,6 +172,7 @@ def _ask_stream_generator(
...
```
Comment (skorokithakis) [inline ez_openai/__init__.py]:
```suggestion
        additional_instructions: str | None = None,
```

**Review by skorokithakis — APPROVED** [review]

File: ez_openai/__init__.py [hunk 4]
```diff
@@ -4,9 +4,11 @@
 from typing import Any
 from typing import Callable
 from typing import Generator
+from typing import Optional
```
Comment (skorokithakis) [inline ez_openai/__init__.py]:
```suggestion
```

**Pull request merged**